app/routes/admin/content/index.js

```diff
@@ -8,5 +8,10 @@ export default Route.extend({
   },
   model() {
     return this.get('store').queryRecord('setting', {});
+  },
+  actions: {
+    willTransition() {
+      this.get('controller.model').rollbackAttributes();
```

Comment:
@niranjan94 That resulted in an error, I didn't knew it was because of that, please check the comments below.